docs/dns-server.md

```diff
@@ -57,10 +80,17 @@ Start menu > Acrylic DNS Proxy > Acrylic UI
 
 * File > Open Acrylic Hosts
 
-Append with your virtual machine IP:
+Append with your virtual machine IP, depends on your network configuration:
 
-```Text
+```bash
+# Redirect *.vm domains to 192.168.56.101 (Other IP, Virtual Machine)
 192.168.56.101 /.*\.vm$
+
+# Redirect *.vm domains to 127.0.0.1 (Local system)
+127.0.0.1 /.*\.vm$
+
+# Redirect *.vm23.iveins.de domains to 192.168.56.101 (Another domain)
+192.168.56.101 /.*\.vm23\.iveins\.de$
```

Comment:
I have this Domain on 127.0.0.1

Reply:
On Windows it depends on how it is configured. As a rule, a virtual machine is taken here. If a host-only adapter is taken, then the IP is 192.168.56.101. If the port is forwarded then 127.0.0.1.

At that time, I decided against port forwarding because I had used port 80 on all systems. In the end, everyone has to know what they prefer.

I've always wondered if a manual would be sent for that, but that would have to be the virtual machine.